test/semantics/doconcurrent05.f90

```diff
@@ -58,7 +56,10 @@ subroutine do_concurrent_test4(n)
   mytest4: if (n>0) then
   mydoc:    do concurrent(j=1:n)
               do concurrent(k=1:n)
+!ERROR: EXIT must not leave a DO CONCURRENT statement
                 if (k==5) exit
+!ERROR: EXIT must not leave a DO CONCURRENT statement
+!ERROR: EXIT must not leave a DO CONCURRENT statement
```

Comment:
Why does this error come out twice?

Reply:
In this test, the ```EXIT``` statement at line 63 causes two different ```DO CONCURRENT``` statements to be exited, one at line 57 and the other at line 58.  It doesn't come out in the test source, but each error message has an additional message attached to it that lists the line being exited.  Thus, the results that the use sees are two different message combinations that point to both errors.

Reply:
Why don't those additional messages appear in the test?

Reply:
Currently, our test infrastructure does not support checking of these additional messages.